### ISSUE TITLE
Plugin pages not working

### ISSUE THREAD

**Issue by diegovogel** [timber/timber#2273]
## Expected behavior
Page templates from plugins continue to work and display content as usual. E.g. Calendar page from The Events Calendar plugin, or Elementor Full Width template from Elementor.

## Actual behavior
Pages that use page templates from plugins are completely blank. Example: Calendar page https://refactor-valor-christian.pantheonsite.io/calendar/

## Steps to reproduce behavior
Install Timber on an existing site and switch theme to the Timber Starter Theme.

## What version of WordPress, PHP and Timber are you using?
WordPress 5.4.2, PHP 7.3, Timber 1.16.0

## How did you install Timber? (for example, from GitHub, Composer/Packagist, WP.org?)
Installed Timber plugin via the WP plugin repo. Then manually installed the Timber Starter Theme.

**Comment by jarednova** [comment]
Hi @diegovogel I tried to replicate the error but was unable to. I used the Events Calendar (WP.org version) and installed it on a site running Timber 1.16. I was able to display the event and the overview page (`/event`)
![Screen Shot 2020-07-09 at 12 14 49 PM](https://user-images.githubusercontent.com/1298086/87064641-004a6f00-c1de-11ea-912c-398150819370.png)

I notice in your description you reference `/calendar`, is there something special going on w your URL structure?
